File: Lecture07/JSONApplication/app/src/main/java/com/example/neera/jsonapplication/MainActivity.java
package com.example.neera.jsonapplication;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.example.neera.jsonapplication.activities.PostActivity;
import com.example.neera.jsonapplication.activities.UsersActivity;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.btnPosts).setOnClickListener(this);
        findViewById(R.id.btnUsers).setOnClickListener(this);
    }

    @Override
    public void onClick(View view) {
        switch (view.getId()) {
            case R.id.btnPosts:
                startActivity(new Intent(this, PostActivity.class));
                break;
            case R.id.btnUsers:
                startActivity(new Intent(this, UsersActivity.class));
                break;
        }
    }
}

File: Lecture12/MediaPlayer/app/src/main/java/com/example/neera/mediaplayer/MainActivity.kt
package com.example.neera.mediaplayer

import android.media.MediaPlayer
import android.net.Uri
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import android.widget.MediaController
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        videoView.setVideoURI(Uri.parse(
                "android.resource://${packageName}/${R.raw.video1}"
        ))
        videoView.start()
        videoView.setMediaController(MediaController(this))

        val mp = MediaPlayer.create(this, R.raw.crowd)

        btnPlay.setOnClickListener(View.OnClickListener { mp.start() })

        btnPause.setOnClickListener(View.OnClickListener { mp.pause() })


    }
}
